pkg/signing/signing.go

```diff
@@ -223,6 +367,70 @@ func SifAddSignature(sinfo *image.Sifinfo, signature []byte) error {
 	return nil
 }
 
+func sykeysFetchPubkey(fingerprint string, sykeysAddr string) (openpgp.EntityList, error) {
```

Comment:
It might make sense to put the stuff that deals with the keyserver in a separate file `sykeys.go`? This seems separate in purpose from actually signing/verifying, which is already quite complex.

Reply:
This is the intention, of course, this will require a complete rework of that code which I'd rather do when all use cases are working simply. Which is about now.

Reply:
Sounds good, feel free to do that in a future PR if it makes more sense.